demo/package.json

```diff
@@ -20,7 +21,13 @@
     "tasit-sdk": "0.0.2"
   },
   "devDependencies": {
+    "babel-eslint": "^10.0.1",
     "babel-preset-expo": "^5.0.0",
+    "eslint": "^5.9.0",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-plugin-prettier": "^3.0.0",
```

Comment:
@marcelomorgado Are both the `eslint-config-prettier` and `eslint-plugin-prettier` required? They may very well be - I forget what the distinction is.

Reply:
@pcowgill `eslint-plugin-prettier` is used to run `prettier` by `eslint` and uses its as rules.
`eslint-config-prettier` avoid conflicts between both disabling `eslint` formatting rules.
More info: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration